includes/admin/emails/filters.php

```diff
@@ -69,4 +69,25 @@ function give_decode_email_tags( $message, $email_obj ) {
 	return $message;
 }
 
-add_filter( 'give_email_message', 'give_decode_email_tags', 10, 2 );
\ No newline at end of file
+add_filter( 'give_email_message', 'give_decode_email_tags', 10, 2 );
+
+/**
+ * This removes the shortcode button from the emails settings page.
+ *
+ * @since 2.2
+ *
+ * @return boolean
+ */
+function give_remove_shortcode_button_on_email_settings_page() {
```

Comment:
@Sidsector9 if possible then do not use core filter to change core functionality. In this case, the code must be in `Give_Shortcode_Button::is_add_button`